### PR TITLE
Removing implementation for initFlow

### PR DESCRIPTION
**Pull request by amanjeetsingh150**
## Proposed Changes

Removing IDB depends on removing pushAppState and pullAppState which are part of deprecated initFlow. This PR aims to remove initFlow implementation defering cleaning up for later

## Testing
- [x] Local
- [x] Staging

### CODE REVIEW

**Review by artem888 — APPROVED** [review]

File: maestro-orchestra/src/test/java/maestro/orchestra/yaml/YamlCommandReaderTest.kt [hunk 1]
```diff
@@ -82,29 +83,6 @@ internal class YamlCommandReaderTest {
         assertThat(e.message).contains("Flow files must contain a config section and a commands section")
     }
 
-    @Test
-    fun initFlow(
-        @YamlFile("007_initFlow.yaml") commands: List<Command>,
```
Comment (artem888) [inline maestro-orchestra/src/test/java/maestro/orchestra/yaml/YamlCommandReaderTest.kt:87]: lets not completely remove those tests , rather test that they throw an exception, you can copy from my PR

File: maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt [hunk 2]
```diff
@@ -612,75 +602,6 @@ class IntegrationTest {
         }
     }
 
-    @Test
-    fun `Case 023 - runFlow with initFlow`() {
```
Comment (artem888) [inline maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt:616]: ditto, keeping one test at least which asserts the correct exception would be nice until we fully deprecate it

File: maestro-orchestra/src/test/java/maestro/orchestra/yaml/YamlCommandReaderTest.kt (hunk 1 above)
Comment (amanjeetsingh150) [reply maestro-orchestra/src/test/java/maestro/orchestra/yaml/YamlCommandReaderTest.kt:87]: Done ✅

File: maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt (hunk 2 above)
Comment (amanjeetsingh150) [reply maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt:616]: Done ✅

**Pull request merged**